installer/mlflow/description.yaml

```diff
@@ -6,7 +6,7 @@ description: |
 namespace: mlflow
 install:
   - type: helm
-    location: https://github.com/fuseml/extensions/raw/main/charts/mlflow-0.0.1.tgz
+    location: https://github.com/fuseml/extensions/raw/release-0.2/charts/mlflow-0.0.1.tgz
```

Comment:
Wait, why 0.2?
We want to point to 0.3 from the main branch, don't we?

Reply:
yes, we do, fixed it :)